Source/ExecutionEngine/ExecutionEngine.cs

```diff
@@ -672,8 +671,8 @@ private Implementation[] GetPrioritizedImplementations(Program program)
         CleanupRequest(requestId);
       }
 
-      if (Options.PrintNecessaryAssumes && program.NecessaryAssumes.Any()) {
-        Console.WriteLine("Necessary assume command(s): {0}", string.Join(", ", program.NecessaryAssumes.OrderBy(s => s)));
+      if (Options.PrintNecessaryAssumes && processedProgram.Program.NecessaryAssumes.Any()) {
+        Console.WriteLine("Necessary assume command(s): {0}", string.Join(", ", processedProgram.Program.NecessaryAssumes.OrderBy(s => s)));
```

Comment:
I don't understand the meaning of the word `processed` here.  What makes a program "processed"?